server/service/src/sync/translations/invoice.rs

```diff
@@ -755,7 +768,9 @@ fn invoice_status(invoice_type: &InvoiceType, data: &LegacyTransactRow) -> Optio
         // inbound
         InvoiceType::InboundShipment | InvoiceType::CustomerReturn => match data.status {
             LegacyTransactStatus::Sg => InvoiceStatus::New,
```

Comment:
Should we not map this one as well? Per OG docs, `sg = suggested (has been opened since first creation, still editable, stock not yet added to mSupply)` ... just because I opened the invoice at one stage, would assume it should still show up as shipped so can edit after migration to OMS 👀

Reply:
> OK POST DIGGING: actually sg is not a thing for SI's, we've removed from docs now... however, if by some munging you end up with an SG transferred invoice, we get the same issue:

I've added a comment for now 2689081422c54281c6aba7d8a2a688d064752407, I hope that's enough